kani-compiler/src/kani_middle/codegen_units.rs

```diff
@@ -0,0 +1,214 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! This module is responsible for extracting grouping harnesses that can be processed together
+//! by codegen.
+//!
+//! Today, only stub / contracts can affect the harness codegen. Thus, we group the harnesses
+//! according to their stub configuration.
+
+// TODO: Move this out of CBMC crate.
```

Comment:
Is this `#TODO` new? Are their any GitHub issues explaining this problem?

Reply:
https://github.com/model-checking/kani/issues/2435

Reply:
Basically interned string is not something specific to CBMC. We added it there a long time ago when dealing with high memory consumption in Kani due to String usage in the CBMC crate.

Reply:
I deleted the TODO